### PR TITLE
Footer Part 2 pull Request

### PR DESCRIPTION
**Pull request by JasonLin43212**

### CODE REVIEW

File: src/js/modules/RoutingApp.js [hunk 1]
```diff
@@ -51,11 +51,13 @@ const RoutingApp = () => {
   return (
     <Provider store={store}>
       <ConnectedRouter history={appHistory}>
+         <PageLayout>
```
Comment (jkao1) [inline src/js/modules/RoutingApp.js]: I've tried this before, and it messes up the routing. Try it; if it's messed up, then delete it. We'll have to figure this out at a later date. (It should just be some arrangement of `ConnectedRouter`, `Switch`, and `PageLayout`.

File: src/js/modules/core/components/PageHeader.js [hunk 2]
```diff
@@ -2,46 +2,12 @@ import React from 'react';
 import injectSheet from 'react-jss';
```
Comment (jkao1) [inline src/js/modules/core/components/PageHeader.js]: add the `// TODO`s I told you about yesterday for the PageHeader into this file

File: src/js/modules/RoutingApp.js (hunk 1 above)
Comment (JasonLin43212) [inline src/js/modules/RoutingApp.js]: Oh shoot I forgot to remove that part. Thanks for pointing it out.

File: src/js/modules/core/components/PageFooter.js [hunk 3]
```diff
@@ -72,7 +87,11 @@ const PageFooter = ({classes, sectionsWithSubsections}) => {
   return (
```
Comment (jkao1) [inline src/js/modules/core/components/PageFooter.js]: you aren't using MUI in all its glory. i like that you figured out flexbox, but definitely use `<Col>`'s also. Moreover, in the design, I've changed column and gutter widths. Which is why declaring `width` in the JSS is a terrible idea. Instead, declare the widths of the `<Col>` and determine gutters by padding between `<Col>`s.

**Comment by NicholasLYang** [comment]
@JasonLin43212 @jkao1  since routing now works in header/footer, can you add it?

**Comment by jkao1** [comment]
the routing is in feature/site-structure-nav LMAO. that's why im rushing to get these two pushed—there's a bit of overlap.

**Comment by NicholasLYang** [comment]
Next time, combine branches. Don't like pushing code that isn't testable by just pulling and running

**Comment by jkao1** [comment]
<3

**Pull request merged**